api/comments/views.py

```diff
@@ -79,6 +79,7 @@ class CommentDetail(JSONAPIBaseView, generics.RetrieveUpdateDestroyAPIView, Comm
         is_abuse       boolean            has this comment been reported by the current user?
         has_children   boolean            does this comment have replies?
         can_edit       boolean            can the current user edit this comment?
+        new_mentions   list               guids of mentioned users in content of comment
```

Comment:
> `new_mentions` is a list of guids of all of the mentioned users that are present when the comment is saved and `old_mentions` is a list of guids of anyone who has already been mentioned in this comment

Sure, but what does `new_comments` mean in the context of a `GET` request? I would expect it to always be empty, since its purpose is to pass new mentions *in* (via a `POST` or `PUT` request), not to get mentions *out*. Right?

I actually would expect `new_mentions` to only exist in `POST`/`PUT` requests from JavaScript to Python, and `old_mentions` to only exist in Python and the database (maybe `ever_mentioned` would be a more descriptive name)?

| Layer | `old_mentions`<br>(&rarr; `ever_mentioned`?) | `new_mentions` |
|:---------|:---------------:|:-----------:|
| API `POST`/`PUT` | ❌ | ✅  |
| API `GET` | ❌  | ❌  |
| Python | ✅  | ✅  |
| database | ✅ | ❌  |